src/common-ssh/sftp.c

```diff
@@ -787,6 +787,13 @@ static int guac_common_ssh_sftp_get_handler(guac_user* user,
     /* Otherwise, send file contents */
     else {
 
+        /* If downloads are disabled, log and return. */
+        if (filesystem->disable_download) {
+            guac_user_log(user, GUAC_LOG_INFO,
+                    "File downloads have been disabled.");
```

Comment:
In line with the other download failure messages (`Unable to read file "..."`, `Unable to read directory "...": (reason)`), I suggest phrasing this message as a failed/refused operation and including the name of the file.

The current message (`File downloads have been disabled.`) is not immediately clear that it indicates that a request isn't being serviced, and could be interpreted as a simple acknowledgement of the download setting.